source/server/connection_handler_impl.h

```diff
@@ -136,13 +136,13 @@ class ConnectionHandlerImpl : public Network::TcpConnectionHandler,
   const absl::optional<uint32_t> worker_index_;
   Event::Dispatcher& dispatcher_;
   const std::string per_handler_stat_prefix_;
+  std::atomic<uint64_t> num_handler_connections_{};
```

Comment:
maybe worth a comment here, to note people won't change this order in the future.

Reply:
Done